### DISCUSSION TITLE
How to exclude header and footer while extracting text?

### DISCUSSION THREAD

**Discussion by Laxmi530** [py-pdf/pypdf#2085] [Q&A] [answered]
Hai,
I am extracting text from pdf file and processing those text, but I noticed that if the pdf file has header and footer in every page, it is including both every time which is not required. So is there any method to skip the header and footer or any method to extract only header and footer from the pdf so that we can replace a empty string in place of header and footer.
Below code is using for text extraction.
```
def pdf_reader(file):
    text = str()
    reader = PdfReader(file)
    for page in reader.pages:
        text += page.extract_text()
    return text
```
so can someone please guide me how to do this.

Thanking you in advance.

**Comment by pubpub-zz** [comment, marked as answer]
https://pypdf.readthedocs.io/en/stable/user/extract-text.html#example-1-ignore-header-and-footer

**Reply by Laxmi530** [reply]
Thanks @pubpub-zz it works.
